### DISCUSSION TITLE
Chase Paymentech

### DISCUSSION THREAD

**Discussion by lancasterjones** [juspay/hyperswitch#4250] [Ideas / Feature Requests]
Hi!
I've been waiting on this connector to jump over to hyperswitch.
Please enable this!

**Comment by gorakhnathy7** [comment]
Hey @lancasterjones!
Thanks for the feature request.
We will get back once we have included in our Roadmap.

**Reply by gorakhnathy7** [reply]
Hey @lancasterjones, Just wanted to understand more about your use case with Chase!
Also, access to the Sandbox environment for Chase Paymentech is a major blocker for us. So, if you already have the integration with it, we can support its integration with hyperswitch.

Looking forward to hear from you.
